### PR TITLE
Sign and deploy steps

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -127,4 +129,96 @@ jobs:
       uses: actions/upload-artifact@v3
       with:
         name: Test Results (${{ matrix.os }} ${{ matrix.configuration }} ${{ matrix.projectloadstyle }})
-        path: test-results/*
\ No newline at end of file
+        path: test-results/*
+        
+  sign:
```
Comment (twsouthwick) [inline .github/workflows/build.yml:134]: this should be a separate workflow that only builds on main or release/ and is triggered by a successful build there

**Review by twsouthwick — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (tomjebo) [reply .github/workflows/build.yml:134]: Yes, I'll separate them out.

**Comment by tomjebo** [comment]
@twsouthwick  another question I had was about the dependencies. It seems deploy should depend on signing which should depend on packing success. Packing should depend on build/test success. 

When I separate these out though it looks like we used manual approval to make sure that sign and deploy were ok to proceed (as well as the release event). I'm not sure how to do approval in github actions, perhaps there's a way?

**Comment by twsouthwick** [comment]
I think for approvals you'll set up a deployment

**Comment by tomjebo** [comment]
sign.yml testing was done from main so this PR is no longer useful. We can submit new PR's for changes to signing and deployment workflows as needed. closing this one.

**Pull request closed without merging**